recipes/LibriSpeech/ASR/transformer/hparams/train_hf_whisper.yaml

```diff
@@ -0,0 +1,139 @@
+# ################################
+# Model: Whisper (Encoder-Decoder) + NLL
+# Augmentation: TimeDomainSpecAugment
+# Authors: Adel Moumen 2022, Titouan Parcollet 2022
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/train_whisper/<seed>
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the biggest Fairseq english whisper model.
+whisper_hub: openai/whisper-tiny
+language: english
+
+# Normalize the english inputs with
+# the same normalization done in the paper
+do_normalize: True
```

Comment:
Variable name not explicit enough (normalized_transcripts) would be better

Reply:
done.